tensorflow/c/kernels/summary_op.cc

```diff
@@ -0,0 +1,170 @@
+
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <sstream>
+
+#include "tensorflow/c/kernels.h"
+#include "tensorflow/c/tf_tensor.h"
+#include "tensorflow/c/kernels/tensor_shape_utils.h"
+#include "tensorflow/core/framework/selective_registration.h"
+#include "tensorflow/core/framework/summary.pb.h"
+#include "tensorflow/core/framework/types.h"
+
```

Comment:
You should include the headers for each symbol that you are using in this file. 

EG: In line 68, we call TF_GetCode, which is declared in tensorflow/c/tf_status.h; we need to include that header directly, and add a build dependency on it as well.

Similarly, there should be inclusions for:
tensorflow/core/platform/protobuf.h due to "SerializeToTstring"
tensorflow/core/platform/logging.h due to "CHECK"
/tensorflow/core/platform/macros.h due to "TF_ATTRIBUTE_UNUSED"
tensorflow/core/platform/tstring.h due to "tstring"
tensorflow/core/platform/strcat.h due to tf::strings::StrCat

etc

Reply:
Thanks, Brian!